python/Integration/Auth0/main.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+from scramjet.streams import Stream
+import requests
+import stripe
+
+url = "https://dev-scp.eu.auth0.com/api/v2/users?per_page=5&sort=created_at%3A1&fields=created_at%2Cupdated_at%2Cemail%2Cnickname&q=user_metadata.newsletter%3D%22true%22"
```

Comment:
Maybe you should add as comment a link to auth0.com docs to explain for end user how this URL was prepared.